src/Tests/MessageUiTest.php

```diff
@@ -119,6 +120,27 @@ public function testMessageTranslate() {
     $this->assertFalse(MessageType::load($type), 'The message deleted via the UI successfully.');
   }
 
+  /*
+   * Test that message render returns message text wrapped in a div.
+   */
+  public function testMessageTextWrapper() {
+    $type = 'dummy_message';
+    // Create message to be rendered.
+    $message_type = $this->createMessageType($type, 'Dummy message', '', array('Text to be wrapped by div.'));
+    $message = Message::create(array('type' => $message_type->id()))
+      ->setAuthorId($this->account->id());
+
+    $message->save();
+
+    // Simulate theming of the message.
+    $build = \Drupal::entityTypeManager()->getViewBuilder('message')->view($message);
+    $output = \Drupal::service('renderer')->renderRoot($build);
+    $this->setRawContent($output);
+
+    $xpath = $this->xpath('//div');
+    $this->assertTrue($xpath, 'A div has been found wrapping the message text.');
```

Comment:
You can just do `strpos("<div>MESSAGE_TEXT", $output) !== FALSE`. Would be much more simple.